radar-upload-frontend/src/axios/file.js

```diff
@@ -69,6 +69,13 @@ export default {
     return axios.put(`/records/${id}/contents/${fileName}`, file, { headers });
   },
 
+  markRecord({ recordId }) {
+    return axios.post(`/records/${recordId}/metadata`, {
+      status: 'READY',
+      revision: 1,
```

Comment:
One thing to note here is the revision won't always necessarily be `1`. Could you please try to process the response received from uploading the content and set the revision return in that response when you mark the record `READY`?

Reply:
ok. What is the revision about?

Reply:
Revision is the way that the record keeps track of edits. A bit like a commit in git.